### PR TITLE
fix: fix k8s styles

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
fix: fix k8s styles

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/config-page/components/table/table.tsx [hunk 1]
```diff
@@ -290,7 +285,7 @@ const BatchOperation = (props: IBatchProps) => {
       <span className="mr-2">{`${i18n.t('selected {xx}', {
         xx: `${selectedRowKeys?.length || 0}${i18n.t('common:items')}`,
       })}`}</span>
-      <Dropdown overlay={dropdownMenu}>
+      <Dropdown overlay={dropdownMenu} zIndex={1000}>
```
Comment (McDaddy) [inline shell/app/config-page/components/table/table.tsx:288]: is `zIndex` a Dropdown api?

Comment (laojun) [reply shell/app/config-page/components/table/table.tsx:288]: yes, `export interface PopconfirmProps extends AbstractTooltipProps`

Comment (laojun) [reply shell/app/config-page/components/table/table.tsx:288]: yes, `export interface PopconfirmProps extends AbstractTooltipProps`

**Review by McDaddy — APPROVED** [review]

**Pull request merged**